.changeset/wise-spoons-bow.md

```diff
@@ -0,0 +1,10 @@
+---
+"@marigold/docs": patch
+"@marigold/components": patch
+"@marigold/theme-b2b": patch
+"@marigold/theme-core": patch
+---
+
+feat(DST-596): add pending indicator in `<Button>` component
```

Comment:
Here it is called _pending indicator_ and in the description _loading indicator_ and _pending state_. I would standardise it everywhere. Also in the code I would use either pending or loading. I think loading is the term we use.

Reply:
what do you mean?  In code the button prop is called "isPending" i replaced this to "pending" and wrote pending everywhere

Reply:
Exactly that. For my understanding we call it normally "loading" everywhere but you called prop and everything pending. Now we have a mix of these words. I would call loading everywhere cause of consistency reasons.

Reply:
yes because the original prop from button was called "isPending" :D Of course I can rename it. @tirado-rx  what do you think?

Reply:
Yes, I agree that we should change it to "loading" everywhere as much as possible. I thought @sebald mentioned this issue today in the daily, but maybe I'm mixing things up.

Reply:
this is the same conversation like in vercel right? xD I think we can close this and discuss there? If not please reopen